src/components/App.tsx

```diff
@@ -1,24 +1,25 @@
 import * as React from 'react'
 import {
-  BrowserRouter as Router,
-  Link,
+  Redirect,
   Route,
   Switch,
-  Redirect
-} from 'react-router-dom'
-import NotFound from './NotFound'
+} from 'react-router'
```

Comment:
where is `react-router` coming from? It's not in the package.json, so if it's included in `react-router-dom` do we need to change this?